build.cake

```diff
@@ -101,6 +119,68 @@ Task("Info")
     Information("Building using {0} version of Cake", cakeVersion);
 });
 
+Task("BuildDocs")
+.IsDependentOn("Build")
+.Does(() =>
+{
+    StartProcess(Context.Tools.Resolve("docfx") ?? Context.Tools.Resolve("docfx.exe"), 
+                 "./docs/docfx.json");
+});
+
+Task("CopyDocsToVersionedDirectories")
+.IsDependentOn("BuildDocs")
+.IsDependentOn("CloneExistingDocs")
+.Does(() =>
+{
+    if (DirectoryExists("./docs/temp/latest"))
+    {
+        DeleteDirectory("./docs/temp/latest", recursive: true);
+    }
+    Information("Copying docs to docs/temp/latest");
+    CopyDirectory("./docs/_site/", "./docs/temp/latest/");
+
+    var travisTag = EnvironmentVariable("TRAVIS_TAG");
+    if (string.IsNullOrEmpty(travisTag))
+    {
+        Console.WriteLine("TRAVIS_TAG not set, won't copy docs to a tagged directory");
+        return;
+    }
+
+    var taggedVersion = travisTag.TrimStart('v');
+    var tagDocsDirectory = string.Format("./docs/temp/{0}", taggedVersion);
+
+    Information("Copying docs to " + tagDocsDirectory);
```

Comment:
Question: Is this a built in Cake method?

Reply:
I took this entire method from the older code, I think here this function is:  https://cakebuild.net/api/Cake.Common.Diagnostics/LoggingAliases/C298F7E8

here is an example of output (9015)
![image](https://user-images.githubusercontent.com/68990515/95750276-9098c400-0ca5-11eb-9172-a9cd7e9c6eaf.png)